tetris/impl/custom.py

```diff
@@ -10,7 +10,10 @@
 from tetris.engine import Queue
 from tetris.engine import RotationSystem
 from tetris.engine import Scorer
-from tetris.impl import modern
+from tetris.impl import gravity
+from tetris.impl import queue
+from tetris.impl import rotation
+from tetris.impl import scorer
```

Comment:
```suggestion
from tetris.impl.gravity import InfinityGravity
from tetris.impl.queue import SevenBag
from tetris.impl.rotation import SRS
from tetris.impl.scorer import GuidelineScorer
```
we only use single objects from these, I think this is less messy